### PR TITLE
changed slow arr to make diff of 0.1 and fast arrow to 1

### PR DESCRIPTION
**Pull request by Vishal0311**
So i have changed the slow arrow to add or subtract 0.1 and fast arrow to add or subtract 1. 
This change can be used if like someone is creating a project and add a horizontal number picker for weight .
So he/she can slide or click the decimal number for his/her weight.

### CODE REVIEW

File: picker/src/main/java/me/angrybyte/numberpicker/view/ActualNumberPicker.java [hunk 1]
```diff
@@ -600,7 +600,7 @@ public void setValue(int newValue) {
      */
     private void onControlClicked(@Control int which) {
         int oldValue = mValue;
-        int changeX = 0;
```
Comment (milosmns) [inline picker/src/main/java/me/angrybyte/numberpicker/view/ActualNumberPicker.java:603]: This is change in pixels, visually, has no effect on the actual stored value.

File: picker/src/main/java/me/angrybyte/numberpicker/view/ActualNumberPicker.java [hunk 2]
```diff
@@ -616,13 +616,13 @@ private void onControlClicked(@Control int which) {
             case FAST_ARR_LEFT: {
                 int valueChange = (mMaxValue - mMinValue) / 10;
                 mValue -= valueChange;
-                changeX = (int) -(0.1f * mWidth);
```
Comment (milosmns) [inline picker/src/main/java/me/angrybyte/numberpicker/view/ActualNumberPicker.java:619]: This hard-codes visual movement to literally 1px, which is probably invisible to the naked eye on high-density displays.

File: picker/src/main/java/me/angrybyte/numberpicker/view/ActualNumberPicker.java [hunk 3]
```diff
@@ -66,8 +66,8 @@
     private Paint mBarPaint;
     private RectF mBarBounds = new RectF(0, 0, 0, 0);
     private int mBarCount = DEFAULT_BAR_COUNT;
-    private int mMinBarWidth = 1;
```
Comment (milosmns) [inline picker/src/main/java/me/angrybyte/numberpicker/view/ActualNumberPicker.java:69]: I think this change is unnecessary as it controls the vertical bar width and doesn't affect the X movement or the stored value.

**Review by milosmns — CHANGES_REQUESTED** [review]
I believe that there was a misunderstanding about this change, I'll give a bit more info below.

There are live usages of this library in real projects, because of which it is not really possible to simply change the functionality the current users are used to. Upgrading to the latest version with this change could potentially break their projects.

The old behavior needs to stay the same, but can be enhanced by the new functionality using some kind of flag. What I had in mind as a potential solution (that doesn't break backwards-compatibility) is introducing a couple of flags as new View attributes:

  - `incrementArrowNormal` : `float` - how much should the **normal** arrow move the value
  - `incrementArrowFast` : `float` - how much should the **fast** arrow move the value
  - `useStaticIncrements` : `boolean` - whether to use the increments or not

With this change, `useStaticIncrements = true` could use the new attributes `incrementArrowNormal` and `incrementArrowFast` to move the value, and `useStaticIncrements = false` would keep the old behavior for the existing library users. 
Therefore, the default value of `useStaticIncrements` needs to be `false`.

Then, after implementing this in the View attributes XML and the custom View, we'd need to update the library version for Gradle, build/test on CI, and upload to jCenter (which is now deprecated), figure out how to push to MavenCentral, prepare release notes and update docs in a couple of places.

That's why I initially said that it's more effort than it seems on the surface. 🙂

And now that I think about it some more... I'm not sure if it would even be possible to push to MavenCentral directly with the current project setup, so even when code is updated - it would be difficult to distribute to end users without huge structural changes.

File: build.gradle [hunk 4]
```diff
@@ -1,9 +1,10 @@
 buildscript {
     repositories {
         jcenter()
+        google()
     }
     dependencies {
```
Comment (yes411) [inline build.gradle:6]: I

File: build.gradle [hunk 5]
```diff
@@ -1,9 +1,10 @@
...
-        classpath 'com.android.tools.build:gradle:2.3.0-beta2'
```
Comment (yes411) [inline build.gradle:6]: > ~~[]()[]()@jorgerego ~~

**Review by yes411 — APPROVED** [review]

**Comment by milosmns** [comment]
Either way, with JCenter shut down, I am unable to release new versions without significant code changes… for which I have no time anymore. Feel free to clone this repo and release on your own if you need the changes.

**Pull request closed without merging**